lib/generators/react_on_rails/base_generator.rb

```diff
@@ -165,7 +165,7 @@ def append_to_assets_initializer
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
           Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
-          Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
+          Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
```

Comment:
It won't be loaded.  You have to add it manually to the precompile list or
include it in the `application.js`.

[image: Inline image 1]

On Mon, Dec 28, 2015 at 4:14 PM, Justin Gordon <notifications@github.com>
wrote:

> In lib/generators/react_on_rails/base_generator.rb
> <https://github.com/shakacode/react_on_rails/pull/172#discussion_r48509314>
> :
>
> > @@ -165,7 +165,7 @@ def append_to_assets_initializer
> >            Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
> >            Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
> >            Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
> > -          Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
> > +          Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
>
> There's many other gems that provide JS files (take jquery-rails, for
> example).
>
> Do they also put them in the precompile list?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shakacode/react_on_rails/pull/172/files#r48509314>.
>